### PR TITLE
(#146) - Support for building el9 packages

### PR DESCRIPTION
**Pull request by gerrity95**
(#146) added support for el9 packages to be built using a template of el8. Also added support for building el9 packages to the nightly build in the github workflow

### CODE REVIEW

File: .github/workflows/nightly.yaml [hunk 1]
```diff
@@ -35,6 +35,21 @@ jobs:
           packager_tag: el8-go1.20
           version: nightly
 
+  el9_64:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          path: src/github.com/choria-io/aaasvc
```
Comment (ripienaar) [inline .github/workflows/nightly.yaml]: lets change this one - and the earlier one by el8 - to `src/github.com/choria-io/stream-replicator` otherwise LGTM

Comment (gerrity95) [reply .github/workflows/nightly.yaml]: Updated now - Also did el7. Can revert that but assumed you wanted all 3 changed

**Review by ripienaar — APPROVED** [review]

**Comment by ripienaar** [comment]
I will do a follow up to update dependencies and move to go 1.21 which will be the minimum @gerrity95

**Comment by ripienaar** [comment]
Screenshot 2024-03-05 at 16 26 56

Nightly build runs, we'll see how it goes tomorrow once its been all the way through and into the repos

**Pull request merged**